components/app/leaderboard/GrantPools.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div>
+    <!-- collapse action -->
+    <div class="divider pb-2">
+      <button
+        type="button"
+        class="btn btn-sm btn-ghost"
+        @click="expand = !expand"
+      >
+        {{ $t('leaderboard.pools') }}
+        <app-icon
+          class="ml-2"
+          :size="24"
+          :data="expand ? icons.caretUp : icons.caretDown"
+        />
+      </button>
+    </div>
+
+    <!-- skeleton loader -->
+    <div
+      v-if="loading && expand"
+      class="grid grid-cols-2 md:grid-cols-3 xl:grid-cols-5 gap-4"
+    >
+      <div
+        v-for="i in 5"
```

Comment:
suggestion(blocking):  I know that at this point we don't have the object to iterate through, but we should at least use a named constant instead of a "magic number".